chaoslib/chaoskube/README.md

```diff
@@ -0,0 +1,3 @@
+note: the chaoskube library is not supported by litmuschaos any more. 
+it will be left here, because it can be used if the chaosServiceAccount has cluster-scope.
+as of version 0.8 chaosexperiments are namespace-scoped
```

Comment:
Can we use some markdown formatting in this README

Reply:
Of course, I just wanted to give some notice. Do you change it?
Also I noticed that I pushed changes in the experiments/generic/pod_delete/pod_delete_k8s_job.yml I did not want to push, I deleted the default values and commented the service account.

Reply:
Cool. Looks good. We could just do something like: 

#### Note
The chaoskube library is not supported by litmuschaos any more.  It will be left here, because it can be used if the chaosServiceAccount has cluster-scope. As of version 0.8 chaosexperiments are namespace-scoped